andela/matterhorn-rc
# #151994111 RESTFUL API endpoints

**Are you submitting to `staging`?**

We currently only accept PR's to `staging` not `master`

- [x] Description explains the issue / use-case resolved
- [ ] Only contains code directly related to the issue
- [ ] Has tests.
- [ ] Has docs.
- [ ] Passes all tests
- [ ] Has been linted and follows the style guide

#### What does this PR do?
Extends Reaction Commerce DDP methods by making RESTful API endpoints corresponding to the methods and publications.

#### Description of Task to be completed?
- run `meteor add simple:rest`, this package exposes the application's publications and methods as RESTFUL API endpoints.

#### How should this be manually tested?
- Pull recent changes into local
- run `npm install` and then run `reaction` from terminal to start up application
- Fire up postman and visit `localhost:3000/publications/api-routes` depending on the host served by your local terminal
- You should see a **JSON** response of all endpoints exposed by the package installed earlier, with corresponding paths and http methods that can be used on them.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#151994111 Make the API endpoints RESTful

#### Screenshots (if appropriate)
<img width="1439" alt="screen shot 2017-10-26 at 9 35 01 am" src="https://user-images.githubusercontent.com/20375577/32042895-142ae53e-ba31-11e7-862e-93abcab1fa70.png">

#### Questions:
N/A
